openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1049,17 +1058,29 @@ The Batch Credential Endpoint issues multiple Credentials in one Batch Credentia
 
 Communication with the Batch Credential Endpoint MUST utilize TLS. 
 
-The Client can request issuance of multiple Credentials of certain types and formats in one Batch Credential Request. This includes Credentials of the same type and multiple formats, different types and one format, or both. 
+The Client can request issuance of multiple Credentials, that may be:
 
-## Batch Credential Request {#batch-credential_request}
+* different Credential Configurations (with possibly different Credential Formats)
+* different Credential Datasets
+* different Credential instances of the same Credential Dataset (with different cryptographic material)
```

Comment:
```suggestion
```suggestion
* different Credential Configurations (with possibly different Credential Formats)
* different Credential Datasets
* different Credential instances of the same Credential Dataset (with different cryptographic material)
```
* different Credential instances of the different Credential Dataset
* different Credential instances of the same Credential Dataset (with different cryptographic material)
```
to me there should be only these two variants

Reply:
you could have the same Credential Dataset with different Credential Configurations, i.e. PID in SD-JWT VC and ISO mdoc